### ISSUE TITLE
Interpolation for feature queries not supported

### ISSUE THREAD

**Issue by xzyfer** [sass/libsass#1400]
``` scss
$query: "(feature1: val)";
$feature: feature2;
$val: val;
@supports (#{$query} and ($feature: $val)) or (not ($feature + 3: $val + 4)) {
  foo {a: b}
}
```

Ruby Sass

``` scss
@supports ((feature1: val) and (feature2: val)) or (not (feature23: val4)) {
  foo {
    a: b; } }
```

LibSass

``` scss
Error: property "#{$query}" must be followed by a ':'
        on line 4 of test.scss
>> @supports (#{$query} and ($feature: $val)) or (not ($feature + 3: $val + 4)) {
```

Spec https://github.com/sass/sass-spec/tree/master/spec/libsass-todo-tests/scss-tests/128_test_supports_with_expressions
##

**Comment by xzyfer** [comment]
I found this issue randomly. Noone has since reported or supported this issue so I don't think it should block 3.4.

This also requires a decent amount of work to do right since we don't yet have an interpolation AST node.

**Comment by xzyfer** [comment]
Fixed by #2572
